site/content/en/docs/Installation/Install Agones/helm.md

```diff
@@ -221,13 +221,11 @@ The following tables lists the configurable parameters of the Agones chart and t
 | `gameservers.podPreserveUnknownFields`                   | Disable [field pruning][pruning] and schema validation on the Pod template for a [GameServer][gameserver] definition                                                                                                    | `false`                            |
 | `helm.installTests`                                      | Add an ability to run `helm test agones` to verify the installation                                                                                                                                                     | `false`                            |
 
-{{% feature publishVersion="1.29.0" %}}
 **New Configuration Features:**
 
 | Parameter                             | Description                                                                                                                                                                                                                           | Default |
 |---------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------|
-| `agones.image.registry`                                  | Global image registry for all the Agones system images                                                                                                                                                                                   | `us-docker.pkg.dev/agones-images/release`
-{{% /feature %}}
+| `agones.image.registry`                                  | Global image registry for all images                                                                                                                                                                                                     | `gcr.io/agones-images` 
```

Comment:
In this one instance, these changes get merged into the table above (and the publishVersion should be incremented to the next release version). This table is here so that we don't have to copy the full table above when we want to add a new row or two.

Reply:
+1 to what @roberthbailey said. These changes need to be merged into the table above, rather than the other way around.